crates/bevy_pbr/src/render/mesh_bindings.wgsl

```diff
@@ -5,11 +5,19 @@
 #ifdef MESH_BINDGROUP_1
 
 @group(1) @binding(0)
-var<uniform> mesh: Mesh;
-
+#ifdef MESH_BATCH_SIZE
+var<uniform> mesh: array<Mesh, #{MESH_BATCH_SIZE}u>;
 #else
+var<storage> mesh: array<Mesh>;
+#endif // MESH_BATCH_SIZE
+
+#else // MESH_BINDGROUP_1
```

Comment:
Nit: It would be clearer if the `@group(1) @binding(0)` was on top of the `var<storage>` instead of on top of the `#ifdef`

```c++
#ifdef MESH__BATCH_SIZE
@group(1) @binding(0)
var<uniform> mesh: array<Mesh, #{MESH_BATCH_SIZE}u>;
#else
@group(1) @binding(0)
var<storage> mesh: array<Mesh>;
#endif // MESH_BATCH_SIZE
```

Reply:
I looked at this and was confused for a moment. I thought I had written it like in your quote and you were suggesting the other direction. Ok, I can duplicate it and put it above the declaration.